timbr/DetailsTableViewController.m

```diff
@@ -24,15 +24,59 @@ - (void)viewDidLoad {
     [super viewDidLoad];
     [self.tableView registerNib:[UINib nibWithNibName:@"DetailsTableViewCell" bundle:nil] forCellReuseIdentifier:@"DetailsTableViewCell"];
     [self.tableView registerNib:[UINib nibWithNibName:@"ChartTableViewCell" bundle:nil] forCellReuseIdentifier:@"ChartTableViewCell"];
+    
+    LogCategory *mockLog = [LogCategory getMockLog];
+    self.logCategory =  mockLog;
+    PFQuery *query = [PFQuery queryWithClassName:@"LogCategory"];
```

Comment:
@ChrisGuzman this code / logic should move inside model. Also why is details fetching whole thing from top. Ideally it should just work way itself from logCategory which collections view passed to this guy.